packages/discovery-react-components/src/utils/useDataApi.ts

```diff
@@ -27,7 +27,8 @@ const dataFetchReducer = (state: any, action: any) => {
       return {
         ...state,
         isLoading: false,
-        isError: true
+        isError: true,
+        error: action.error || null
```

Comment:
i think for `FETCH_SUCCESS` and `FETCH_REQUEST` we might want to reset `error` to `null` otherwise it could linger around from previous requests

Reply:
good point, done!